README.md

```diff
@@ -5,6 +5,10 @@ linux: [![Build Status](https://travis-ci.org/ros-controls/ros2_control.svg?bran
 
 ## Disclaimer
 
+`ros2_control` is under heavy development, to get an update of the project please either tag along to one of our working group meetings (see more on discourse) and/or check the resources below:
```

Comment:
perhaps add the full link to discourse in case they aren't aware:  `discourse.ros.org`

Reply:
good point, added!